src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1434,7 +1446,7 @@
 examples:
   - name: Delete a managed application. (autogenerated)
     text: |
-        az managedapp delete --name MyManagedApplication --resource-group MyResourceGroup
+        az managedapp delete --name MyManagedApplication --resource-group MyResourceGroup --subscription mysubscription
```

Comment:
```suggestion
        az managedapp delete --name MyManagedApplication --resource-group MyResourceGroup --subscription MySubscription
```
Can the style of upper and lower case for the sample values be consistent?

Reply:
It's a pain point that Aladdin hasn't been solving for a long time